examples/get-started/first_figure.py

```diff
@@ -0,0 +1,159 @@
+# To see the figure, call :meth:`pygmt.Figure.show`. If you are using a Python
+# script instead of a notebook, use ``fig.show(method="external)`` to display
+# the figure.
+
+fig.show()
+
+###############################################################################
+# Color the land and water
+# ------------------------
+#
+# This figure plots all of the coastlines in the given region, but it does not
+# indicate where the land and water are. Color values can be passed to ``land``
+# and ``water`` to set the colors on the figure.
+#
+# When plotting colors in PyGMT, there are multiple
+# :gmt-docs:`color codes <gmtcolors.html>`, that can be used. This includes
+# standard GMT color names (like ``skyblue``), R/G/B levels (like ``0/0/255``),
+# a hex value (like ``#333333``), and a graylevel (like ``50``). For this
+# example, GMT color names are used.
+
+fig = pygmt.Figure()
+fig.coast(
+    region=[-69, -68, 43.75, 44.75],
+    shorelines=True,
+    land="lightgreen",
+    water="lightblue",
+)
+fig.show()
+
+###############################################################################
+# Set the projection
+# ------------------
+#
+# This figure now has its colors set, but there is not projection or size
+# set for the map. Both of these values are set using the ``projection``
+# parameter.
+#
+# The appropriate projection varies for the type of map. The available
+# projections are explained in the :doc:`projection </projections/index>`
+# gallery. For this example, the Mercator projection is set using ``"M"``.
+# The width of the figure will be 10 centimeters, as set by ``"10c"``. The map
+# size can also be set in inches using "i" (e.g. a 5 inch wide Mercator
+# projection would use ``"M5i"``).
+
+fig = pygmt.Figure()
+fig.coast(
+    region=[-69, -68, 43.75, 44.75],
+    shorelines=True,
+    land="lightgreen",
+    water="lightblue",
+    projection="M10c",
+)
+fig.show()
+
+###############################################################################
+# Add a frame
+# -----------
+#
+# While that the map's colors, projection, and size have been set, the region
+# that is being displayed is not apparent. A frame can be added to
+# annotate the latitude and longitude of the region.
+#
+# The ``frame`` parameter is used to add a frame to the figure. For now, it
+# will be set to ``a`` to use default settings, but later tutorials will
+# show how ``frame`` can be used to customize the axes, gridlines, and titles.
+
+fig = pygmt.Figure()
+fig.coast(
+    region=[-69, -68, 43.75, 44.75],
+    shorelines=True,
+    land="lightgreen",
+    water="lightblue",
+    projection="M10c",
+    frame="a",
+)
+fig.show()
+
+###############################################################################
+# Additional exercises
+# --------------------
+#
+# This is the end of the first tutorial. Here are some additional exercises
+# for the concepts that were discussed:
+#
+# 1. Make a map of Germany using its DCW ISO code ("DE"). Pass the ISO code
```

Comment:
> DCW ISO code

I think there is no "DCW ISO". There is only "ISO country code".

Reply:
fixed in dfc996d